android/modules/platform/src/java/ti/modules/titanium/platform/PlatformModule.java

```diff
@@ -228,7 +237,44 @@ public boolean openURL(String url)
 	public String getMacaddress()
 	// clang-format on
 	{
-		return TiPlatformHelper.getInstance().getMacaddress();
+		//		String macaddr = null;
```

Comment:
We should deprecate this, on Android 6.0+ the MAC address will always return `02:00:00:00:00:00`